include/IECorePython/ParameterBinding.h

```diff
@@ -105,10 +105,20 @@ class ParameterWrapper : public IECorePython::RunTimeTypedWrapper<T>
 						return boost::python::extract<bool>( r[0] );
 					}
 				}
+
+#ifdef _MSC_VER
+				#pragma warning( disable:4101 )
+#endif
```

Comment:
I would prefer to have as few of these pragmas in the code as possible - they just interrupt the flow. Can we just catch `error_already_set &` instead of `error_already_set &e`?

Reply:
Much better! Fixed in https://github.com/ImageEngine/cortex/pull/1217/commits/99c705175a947f73b3c3655b3cd221ed5ad5a5db